radio/src/opentx.cpp

```diff
@@ -1660,21 +1660,31 @@ void instantTrim()
   evalInputs(e_perout_mode_notrainer);
 
   for (uint8_t stick=0; stick<NUM_STICKS; stick++) {
-    if (stick!=THR_STICK) {
-      // don't instant trim the throttle stick
-      uint8_t trim_phase = getTrimFlightMode(mixerCurrentFlightMode, stick);
-      int16_t delta = 0;
-      for (int e=0; e<MAX_EXPOS; e++) {
-        ExpoData * ed = expoAddress(e);
-        if (!EXPO_VALID(ed)) break; // end of list
-        if (ed->srcRaw-MIXSRC_Rud == stick) {
-          delta = anas[ed->chn] - anas_0[ed->chn];
+
+    if (stick!=THR_STICK) { // don't instant trim the throttle stick
+      bool addTrim = true;
+      for (uint8_t i=0; i<MAX_INPUTS; i++) {
```

Comment:
why MAX_INPUTS and not MAX_EXPOS (agree that the name is not very nice)?

Reply:
Yes, better MAX_EXPOS